tests/t_pdxinfo.nim

```diff
@@ -31,6 +31,9 @@ suite "Pdxinfo generation":
             imagePath=launcher
```

Comment:
Should there be a testcase for a minimal pdx where only non-optional fields are present? Or is that covered?

Reply:
Good call. I'll add one

Reply:
Actually, looks like we've already got one! https://github.com/samdze/playdate-nim/pull/105/files/ed9b994e7f8618c6713d831f57f5adaa43febf7f#diff-32b65e69957b6f7d0e4864481eec381cb152fcea9088db2f92068a8b9c2fdc3eR5-R23